dev-python/shellingham/shellingham-1.5.0.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Tool to Detect Surrounding Shell"
+HOMEPAGE="
+	https://github.com/sarugaku/shellingham/
+	https://pypi.org/project/shellingham/
+"
+SRC_URI="$(pypi_sdist_url)"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="test? ( dev-python/pytest-mock[${PYTHON_USEDEP}] )"
```

Comment:
Always wrap and indent, please, i.e.

```
BDEPEND="
    test? (
        ....
    )
"

(ofc tab indent, not spaces like I do because gh xP)

Also `BDEPEND` for Python packages.